### PR TITLE
Update kubeflow versioning policies table 1.4

### LINKED ISSUE #3000: Update Kubeflow versioning policies for 1.4
https://www.kubeflow.org/docs/reference/version-policy/#kubeflow-application-matrix

The table on the versioning policies page needs to be updated to match the table here:
https://github.com/kubeflow/manifests

### PR DESCRIPTION
**Pull request by shannonbradshaw**
fixes #3000

### CODE REVIEW

File: content/en/docs/reference/version-policy.md [hunk 1]
```diff
@@ -112,56 +112,55 @@ documentation for that application.
...
+        <td>v0.6.0</td>
       </tr>
       <tr>
         <td><a href="/docs/components/training/mpi/">MPI training: MPI 
           operator</a>
           (<a href="https://github.com/kubeflow/mpi-operator">GitHub</a>)
         </td>
         <td>Alpha</td>
-        <td></td>
+        <td>v0.3.0</td>
```
Comment (andreyvelich) [inline content/en/docs/reference/version-policy.md:130]: I updated API versions in this PR: https://github.com/kubeflow/website/pull/3002, some of the components became stable.
@shannonbradshaw Is it fine for you ?

**Comment by shannonbradshaw** [comment]
@kimwnasptd I addressed your comment. Please have a look.

File: content/en/docs/reference/version-policy.md [hunk 2]
```diff
@@ -153,37 +153,36 @@ documentation for that application.
...
-          Controller for multi-user isolation</a> 
+        <td><a href="/docs/components/multi-tenancy/">Profile Controller for multi-user isolation</a> 
           (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">GitHub</a>)
         </td>
         <td>Stable</td>
-        <td>1.0.0</td>
+        <td>v1.4.0</td>
       </tr>
       <tr>
         <td><a href="/docs/external-add-ons/serving/seldon">Seldon Core Serving</a>
```
Comment (zijianjoy) [inline content/en/docs/reference/version-policy.md:188]: Can you update `Kubeflow Pipelines SDK` down in this page to be 1.7.0 too?

Comment (shannonbradshaw) [reply content/en/docs/reference/version-policy.md:188]: @zijianjoy done!

**Pull request merged**